src/statmodels.jl

```diff
@@ -216,13 +216,15 @@ Supported variants are:
 - `:MacFadden` (a.k.a. likelihood ratio index), defined as ``1 - \\log (L)/\\log (L_0)``;
 - `:CoxSnell`, defined as ``1 - (L_0/L)^{2/n}``;
 - `:Nagelkerke`, defined as ``(1 - (L_0/L)^{2/n})/(1 - L_0^{2/n})``.
+- `:deviance_ratio`, defined as ``1 - D/D_0``.
```

Comment:
I'd avoid the underscore for consistency with other variants:
```suggestion
- `:devianceratio`, defined as ``1 - D/D_0``.
```